pkgs/development/libraries/gensio/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, lib, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "gensio";
+  version = "1.3.3";
+
+  src = fetchurl {
+    url = "https://downloads.sourceforge.net/sourceforge/ser2net/${pname}-${version}.tar.gz";
```

Comment:
Consider using github repository: https://github.com/cminyard/ser2net

Reply:
Done, thanks for the review.